CHANGELOG.md

```diff
@@ -43,6 +47,7 @@ _ `_optional` subpackage for managing optional dependencies
   and guidelines from [Scientific Python](https://scientific-python.org/specs/spec-0000/)
 
 ### Fixed
+- Added missing `SubspaceContinuous.from_product` constructor
```

Comment:
imo this is added and not fixed

Reply:
had the same in mind originally. not sure why I put it to `fixed`. have updated it now